app/views/subnets/index.html.erb

```diff
@@ -7,7 +7,7 @@
   <thead>
     <tr>
       <th class="col-md-3"><%= sort :name, :as => s_("Subnet|Name") %></th>
-      <th class="col-md-2"><%= sort :network, :as => s_("Subnet|Network") %></th>
+      <th class="col-md-2"><%= sort :network, :as => s_("Subnet|Network Address") %></th>
       <th class="col-md-3"><%= _("Domains") %></th>
```

Comment:
@domcleal what you meaning is if want to update the strings, fixing the capitalization problem, I need to update the English translation file: locale/en/foreman.po, Is that right?